### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bfolly88**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,84 @@
...
+Get twitter API access and put bearer token in a `.env` file in the repository root e.g.
+BEARER_TOKEN=abcdefghijklmnop
+
+
+Due to twitter API restrictions, it is advised to run `twitter.py` for an extended amount of time by editing the last line in the file for the desired twitter handle and date and running the following for as long as data is desired for:
+$ watch -n240 twitter.py
+
+
+Next, simply run the following
+$ python main.py
```
Comment (ammo888) [inline README.md:82]: This should be surrounded line above and below with ``` to make it a code snippet - should look similar to how it is in google doc. This applies to these lines as well:
BEARER_TOKEN=abcdefghijklmnop
$ watch -n240 twitter.py

File: README.md [hunk 2]
```diff
@@ -1 +1,84 @@
...
+
+## **Project Overview**
+The premise of this project is to explore the relationship between twitter activity and a potentially related stock. Specifically, we aim to answer if #there is a lead-lag correlation between sentiment of twitter mentions of a user and some stock price. The end goal is a tool that takes in a twitter #handle, stock ticker, and day of interest, performs sentiment analysis on tweet mentions, and correlates aggregated sentiment against the stock price #movements.
+
+#Limitations and Future Enhancements
+	Machine learning is a complex set of algorithms and functions that rely on clean and precise data inputs. We spend a lot of time cleaning and training data so that when input into a machine learning library or model, we get the output we are expecting. Twitter data is no exception to being problematic as an input into our spaCy models. Tweets can range from being concise professional language to slang and even other machine learning robots’ text that doesn't really resemble a coherent language. With this reality, We understand that there is a higher degree of noise in our ‘tweet’ data simply because of the nature of tweets and internet gargon.
```
Comment (ammo888) [inline README.md:15]: Remove indent

File: README.md [hunk 3]
```diff
@@ -1 +1,84 @@
...
+
+  Another limitation we faced when working on this project was the number of allowed tweets as well as the availabilty of twitter data fo revery corresponding stock price. There is much more consistnet and non-varying data when it comes to stock price data, compared to tweets. so, to combat this reality we broadned out resolution to aggregate tweets daily, insteal of 5 minutes or some other smaller time frame.
```
Comment (ammo888) [inline README.md:17]: Remove indent

File: README.md [hunk 4]
```diff
@@ -1 +1,84 @@
-# twitter-stock-analysis
\ No newline at end of file
+
+
+## Twitter Stock Analysis
+
+## **Our Team/ Our Responsibilities**
```
Comment (ammo888) [inline README.md:5]: Anywhere there is ## you don't need ** to make it bold

File: README.md [hunk 5]
```diff
@@ -1 +1,84 @@
-# twitter-stock-analysis
\ No newline at end of file
+
+
+## Twitter Stock Analysis
```
Comment (ammo888) [inline README.md:3]: Change to #

File: README.md [hunk 6]
```diff
@@ -1 +1,84 @@
...
+## **Our Team/ Our Responsibilities**
+
+Minh - managing user input, data caching, analysis pipeline, twitter sentiment vs. stock price analysis
+Bryan - data fetching, cleaning + preparation for tweets and stock prices
+Elliott- sentiment analysis on tweets, extracting features, and presentation
+
+## **Project Overview**
+The premise of this project is to explore the relationship between twitter activity and a potentially related stock. Specifically, we aim to answer if #there is a lead-lag correlation between sentiment of twitter mentions of a user and some stock price. The end goal is a tool that takes in a twitter #handle, stock ticker, and day of interest, performs sentiment analysis on tweet mentions, and correlates aggregated sentiment against the stock price #movements.
+
+#Limitations and Future Enhancements
```
Comment (ammo888) [inline README.md:14]: Use ## and add space between the # and the words

File: README.md (hunk 4 above)
Comment (ammo888) [reply README.md:5]: Same applies to rest of this file

File: README.md [hunk 7]
```diff
@@ -1 +1,84 @@
...
+
+## **Sentiment Analysis**
+
+We are using a machine learning tool called sentiment analysis in our project. This tool will analyze the text data that we feed it and compute a polarity metric which we will call a ‘sentiment score’. The main library used is called spaCy and the additional libraries (pipelines) are Asent, and SpacyTextBlob.
+
+Spacy is a free open source library used for NLP (Natural Language Processing) and Asent and SpacyTextBlob are rule-based sentiment analysis tools which was developed on top of spaCy as the underlying library. These pipelines have additional features that can be used for further analytics, but we are just interested in the polarity.	
+
+Our goal in this module is to apply these pipelines to the twitter data and give every tweet a basic sentiment score that we will then correlate with the stock data.
+
+spaCy: https://spacy.io/
```
Comment (ammo888) [inline README.md:37]: For links like this, use this syntax: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

Applies to rest of this file

File: README.md [hunk 8]
```diff
@@ -1 +1,84 @@
...
+
+Each of those steps are run independently of each other, requiring no explicit interaction between the methods. At the data processing step, caching based on existing CSV files is used. If there’s already stock price data for a given stock ticker and date in the directory, stock data fetching is skipped. Similarly, if twitter tweets for a given handle and date are present, fetching tweets is also skipped. For sentiment analysis, performing analysis is similarly skipped if the data exists. This caching is to avoid repeatedly invoking expensive and lengthy operations.
+
+
+## **Setup**
+
+Environment
+* `$ conda create -n fintech python==3.7.2`
+* `conda activate fintech`
+* `pip install -r requirements.txt`
```
Comment (ammo888) [inline README.md:69]: Instead of bullet points here, just surround entire thing in a code block using ```

**Review by itselliott — COMMENTED** [review]
Reviewed

**Review by itselliott — APPROVED** [review]
Reviewed

**Pull request merged**